src/server.js

```diff
@@ -36,10 +36,17 @@ module.exports = function() {
 		server = server.createServer(app).listen(config.port, config.host);
 	} else {
 		server = require("spdy");
-		server = server.createServer({
-			key: fs.readFileSync(Helper.expandHome(config.https.key)),
-			cert: fs.readFileSync(Helper.expandHome(config.https.certificate))
-		}, app).listen(config.port, config.host);
+		let keyPath = Helper.expandHome(config.https.key);
+		let certPath = Helper.expandHome(config.https.certificate);
+		if (fs.existsSync(keyPath) && fs.existsSync(certPath)) {
+			server = server.createServer({
+				key: fs.readFileSync(keyPath),
+				cert: fs.readFileSync(certPath)
+			}, app).listen(config.port, config.host);
+		} else {
```

Comment:
I would prefer two separate messages for each of the files. And do a negative check, then you dont need an `else`.